worker/backend/libcontainerd/client.go

```diff
@@ -107,3 +109,6 @@ func (c *client) Version(ctx context.Context) (err error) {
 	_, err = c.containerd.Version(ctx)
 	return
 }
+ func (c *client) Destroy(ctx context.Context, handle string) (err error) {
+ 	return
```

Comment:
Does this return work? We declared an error type to return, but this line will return nothing.

Reply:
yep, it's a no-op - from the commit message:

> - still need to actually invoke containerd's Destroy and add integration
tests

the current PR has only the high-level logic w/out the actual functionality in place yet 😁